### ISSUE TITLE
Use a different proxy URL with jolokia for vanilla k8s

### ISSUE THREAD

**Issue by gashcrumb** [hawtio/hawtio-kubernetes#93]
Which should hopefully sort [this issue](https://github.com/fabric8io/fabric8/issues/4646)

**Comment by jimmidyson** [comment]
FYI this isn't a Kubernetes issue per se but a GKE specific issue to do with routing over SSH tunnels.

**Comment by gashcrumb** [comment]
@jimmidyson ah, k.  Think this would still work as a workaround at least?

**Comment by jimmidyson** [comment]
Yeah it should do, just make sure not to affect OpenShift as otherwise security stuff will be borked. As soon as we can move back to a consistent way of doing it we should.

**Comment by gashcrumb** [comment]
Yep, will do, it's an either/or thing so should be fine...
